documentation/docs/pages/docs/start-building/add.mdx

```diff
@@ -26,11 +26,14 @@ it then ask you 2 questions:
 networks:
   - name: ethereum // [!code focus]
     chain_id: 1
+    kind: Rpc
```

Comment:
i wouldnt change the rindexer yaml i would work this out under the hood basically if rpc .includes hypersync then you know its turned on

Reply:
i could make default kind = Rpc, so we won't change default rindexer yaml. But considering other impls in the future (exex, sqd) it better to specify it trough param

Reply:
yeah ok we can make it default RPC and then not add it to all examples but just have 1 example of how to enable it for hypersync?